core/src/utils/test/playwright/page/utils/goto.ts

```diff
@@ -78,10 +78,18 @@ configs().forEach(({ config, title }) => {
     description: formattedRtl === 'true' ? 'rtl' : 'ltr',
   });
 
-  const result = await Promise.all([
-    page.waitForFunction(() => (window as any).testAppLoaded === true, { timeout: 4750 }),
-    originalFn(formattedUrl, options),
-  ]);
+  const res = await originalFn(formattedUrl, options);
 
-  return result[1];
+  /**
+   * Set the direction before the tests start
+   * to avoid https://bugs.webkit.org/show_bug.cgi?id=257133
```

Comment:
I need to add a tech debt ticket to remove this once the bug has been fixed/released in Safari and Playwright has been updated.